engine/src/main/java/com/datatorrent/stram/LaunchContainerRunnable.java

```diff
@@ -172,6 +173,18 @@ public void run()
         if (archives != null) {
           addFilesToLocalResources(LocalResourceType.ARCHIVE, archives, localResources, fs);
         }
+        String configuredAppPath = dag.getValue(LogicalPlan.APPLICATION_PATH);
+        Path log4jPath = new Path(configuredAppPath + File.separator + StramClient.APEX_LOG4J_PROPS_FILE);
```

Comment:
If the user changes the JVM option (-Dlog4j.configuration=log4j.props) using a different name, how would this code use that value? Aren't you using the hardcoded log4j.props name?